### ISSUE TITLE
AutoStationConfig broken with latest patch

### ISSUE THREAD

**Issue by randomlypickedname** [Pasukaru/DSP-Mods#6]
```
Exception hit 524 times: MissingMethodException: Method not found: 'PlanetTransport.SetStationStorage'.
Pasukaru.DSP.AutoStationConfig.PlanetTransportPatch.NewStationComponent (PlanetTransport,StationComponent) <0x00168>
(wrapper dynamic-method) PlanetTransport.DMD<PlanetTransport..NewStationComponent> (PlanetTransport,int,int,PrefabDesc) <0x00781>
(wrapper dynamic-method) PlanetFactory.DMD<PlanetFactory..CreateEntityLogicComponents> (PlanetFactory,int,PrefabDesc,int) <0x0221a>
PlanetFactory.AddEntityDataWithComponents (EntityData,int) <0x00195>
(wrapper dynamic-method) PlanetFactory.DMD<PlanetFactory..BuildFinally> (PlanetFactory,Player,int) <0x004ec>
MechaDroneLogic.Build (int) <0x00075>
MechaDroneLogic.UpdateDrones (single) <0x001c7>
MechaDroneLogic.GameTick (long,single) <0x000fc>
Mecha.GameTick (long,single) <0x00043>
Player.GameTick (long) <0x000e6>
(wrapper dynamic-method) GameData.DMD<GameData..GameTick> (GameData,long) <0x0013d>
GameMain.FixedUpdate () <0x00262>

Target methods matching by name:
StationComponent PlanetTransport::NewStationComponent(System.Int32 _entityId, System.Int32 _pcId, PrefabDesc _desc)
System.Void PlanetFactory::CreateEntityLogicComponents(System.Int32 entityId, PrefabDesc desc, System.Int32 prebuildId)
System.Void PlanetFactory::BuildFinally(Player player, System.Int32 prebuildId)
System.Void GameData::GameTick(System.Int64 time)

Relevant plugins:
1. pasukaru.dsp.AutoStationConfig - Pasukaru-AutoStationConfig\AutoStationConfig.dll
	Prefix[index=0]: static System.Void Pasukaru.DSP.AutoStationConfig.PlanetTransportPatch::NewStationComponent(PlanetTransport __instance, StationComponent __result)

==================
```
Occurs when placing a new ILS.

**Comment by RJWoodhead** [comment]
Confirmed.

**Comment by wasserts** [comment]
Same crash here.
It seems like brokenmass (Multibuild Mod) had the same issue and found a quick fix. Maybe you can ask him about it on the DSP Discord.

**Comment by Pasukaru** [comment]
Fixed with v1.3.1
https://dsp.thunderstore.io/package/Pasukaru/AutoStationConfig/
